chore(table): update table tutorial to be more easily understandable (#224)
patternfly/training-scenarios: react-components/table-intro/step1.md

}
</pre>

> So, here are two ways you could write the same columns definition:

Reviewer: ```suggestion
Here are two ways that the same column definition can be written:
```
